backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1650,3 +1650,69 @@ databaseChangeLog:
                   defaultValueNumeric: 200
                   constraints:
                     nullable: false
+  - changeSet:
+      id: add-patient-preferences
+      author: adam@skylight.digital
+      comment: Add table for non-demographic preferences relating to a Person
+      changes:
+        - createTable:
+            tableName: patient_preferences
+            columns:
+              - column:
+                  name: internal_id
+                  type: *idtype
+                  remarks: The internal database identifier of the Person to whom these preferences belong
+                  constraints:
+                    foreignKeyName: fk__patient_preferences__person
+                    references: person
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: preferred_language
+                  type: *string
+                  remarks: Tracks the individual's preferred language
+              - column:
+                  name: test_result_delivery
+                  type: ${database.defaultSchemaName}.TEST_RESULT_DELIVERY
+                  remarks: Tracks the individual's preference for receiving test results
+              - column: *created_at_column
+              - column: *created_by_column
+              - column: *updated_at_column
+              - column: *updated_by_column
+  - changeSet:
+      id: populate-patient-preferences
+      author: adam@skylight.digital
+      comment: Migrate person.test_result_delivery to into patient_preferences
+      changes:
+        - sql:
+          sql: |
+            WITH migration_user as (
+              SELECT internal_id as migrations_user_id
+              FROM ${database.defaultSchemaName}.api_user
+              WHERE login_email = '${migrations_user_email}')
+            INSERT INTO ${database.defaultSchemaName}.patient_preferences (
+                internal_id, preferred_language, test_result_delivery,
+                created_at, created_by, updated_at, updated_by)
+            SELECT
+              person.internal_id internal_id,
+              NULL preferred_language,
+              test_result_delivery,
+              now() created_at,
+              migrations_user_id created_by,
+              now() updated_at,
+              updated_by
```

Comment:
I know I asked for this, but I think I was tired and spitballing—it probably makes more sense to use `migrations_user_id` as both created-by and updated-by since we're not sure who created or updated the `test_result_delivery` column. The only reason it would make sense to copy the values from `person` is if we want to make some claim about the modification times/users of the person record and how they map to modifications of this new record, and I don't think that is worth the trouble. (I was probably thinking back to the multi-facility migration, where it made sense to attribute the creation and modification times of the organization to the facility, since we were splitting the table vertically and most of the interesting bits were going to the new table.)

Reply:
Gotcha, no worries! I'll set this back to `migrations_user_id`